### DISCUSSION TITLE
unused GPIO pin for DS18x20 temperature sensors?

### DISCUSSION THREAD

**Discussion by dalklein** [emporia-vue-local/esphome#56] [Ideas]
Is there any reason I shouldn't be able to use an unused GPIO pin to log DS18x20 temperature sensors?   There's ESPhome code for the Dallas DS18x20 sensors, and I guess there must be an open unused pin on the ESP32 in the Vue2.     Right near my electrical panel are a number of hvac items that I'd like to be logging temperatures and using for controls.    

I have a Vue2 that I have not set up yet, but I'll start with converting it to ESPhome.
Thanks!

**Comment by flaviut** [comment]
No technical reason. The low voltage side is isolated from the high voltage side.

If you perform this mod though, your device will no longer be UL listed. That's important to me, it may not be to you. I would personally prefer to spend the $15 to have a separate device and keep the UL listing intact.

**Reply by dalklein** [reply]
True.  I was considering the wisdom of adding the wire coming out of the panel box and the possibility of it getting energized.
